workspaces/util/src/chunkBySize.ts

```diff
@@ -1,7 +1,15 @@
 export function* chunkBySize<T>(
-  array: readonly T[],
+  array: readonly T[] | null | undefined,
```

Comment:
I believe @elimanzo meant an array that _contains_ null or undefined values. In particular, it could be interesting to have a test using [sparse arrays](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Indexed_collections#sparse_arrays).

We shouldn't relax the TypeScript type here to include null or undefined, and we also can skip the explicit check in the code.

Reply:
Thanks for the feedback! I've implemented the suggested approach to handle sparse arrays and undefined elements safely. The updated code has been pushed.